efficientdet/third_party/grad_checkpoint/memory_saving_gradients.py

```diff
@@ -0,0 +1,519 @@
+"""Recompute tensorflow graph reducing GPU memory usage."""
+import logging
+import contextlib
+import time
+import sys
+from toposort import toposort
+import numpy as np
```

Comment:
Is it possible to replace this toposort with a function?  (We don't have this library internally).

Reply:
Ok. Done

Reply:
Ok. Done